addons/resource/models/resource_calendar.py

```diff
@@ -101,12 +101,14 @@ def default_get(self, fields):
     tz_offset = fields.Char(compute='_compute_tz_offset', string='Timezone offset')
     two_weeks_calendar = fields.Boolean(string="Calendar in 2 weeks mode")
     two_weeks_explanation = fields.Char('Explanation', compute="_compute_two_weeks_explanation")
+    flexible_hours = fields.Boolean(string="Flexible Hours", default=False, help="If checked, employee will be able to change their working hours.")
```

Comment:
by default boolean field is already False

Reply:
Good to know, thank you! I will remove the default value